server/simulation/random.js

```diff
@@ -32,6 +32,14 @@ const randomRating = () => {
   return parseFloat(rating);
 };
 
+/**
+ *   
+ * Returns an object with two properties: one number between 4 and 89, and one number between 2 and 9 
+ * 
+ * @returns {object} with 2 properties,
```

Comment:
Describe the returned object using the same syntax as https://github.com/Rob-Rychs/missioncontrol/blob/db2e4c2891281c4a2837cd9aeac8c909d3686c37/server/simulation/random.js#L57